src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -55,6 +54,14 @@ export const ArtworkApp: React.SFC<Props> = props => {
         </Row>
       )}
 
+      <Row>
+        <Col>
+          <Box mt={6}>
+            <OtherWorks artworkID={props.artwork.id} />
```

Comment:
I think a better component organization might be:

- the `ArtworkApp` component (this one), queries for the context/__typename stuff. As in, this component will have the data about what 'type' of work this is.
- `OtherWorks` _doesnt_ need to do the context/__typename stuff, and instead would receive a prop like: `auctionWork={true}`, or `fairWork={true}`.
- Then, `OtherWorks` can in a straightforward fashion request _just_ the grids that it needs, based on that incoming prop.

Reply:
@mzikherman - this is still very WIP, but the idea (not yet complete in the PR) is that from within `OtherWorks` there are sub-components that group the grids depending on the context and only load the data / components that are needed (this relates to [following comment](https://github.com/artsy/reaction/pull/1572/files/1b50ae8cc14ca7a9672eca73e05a59938aecebf4#diff-ec51be77b425750451e8872c83d78835) as well.) 

It makes sense to hoist the `__typename` up to `ArtworkApp`, though 👍 will do.